pre_vs_berkeley.md

```diff
@@ -0,0 +1,440 @@
+| Reaction                            |                                            |                           |
+| ReactionParticipant                 |                                            |                           |
+| ReadBasedTaxonomyAnalysisActivity   | ReadBasedTaxonomyAnalysis                  | WorkflowExecutionActivity |
+| ReadQcAnalysisActivity              | ReadQcAnalysis                             | WorkflowExecutionActivity |
+| Solution                            |                                            |                           |
+| SolutionComponent                   |                                            |                           |
+| UrlValue                            |                                            | AttributeValue            |
+| WorkflowExecutionActivity           |                                            |                           |
+|                                     | CalibrationInformation                     |                           |
+|                                     | ChemicalConversionProcess                  |                           |
+|                                     | ChromatographyConfiguration                |                           |
+|                                     | Configuration                              |                           |
+|                                     | DissolvingProcess                          |                           |
+|                                     | InformationObject                          |                           |
+|                                     | Instrument                                 |                           |
+|                                     | MassSpectrometry                           |                           |
+|                                     | MassSpectrometryConfiguration              |                           |
+|                                     | MaterialProcessing                         |                           |
+|                                     | MobilePhaseSegment                         |                           |
+|                                     | NucleotideSequencing                       |                           |
+|                                     | PortionOfSubstance                         |                           |
+|                                     | ProtocolExecution                          |                           |
+|                                     | StorageProcess                             |                           |
+|                                     | WorkflowExecution                          |                           |
+
+For example, `MetagenomeAnnotationActivity` used to be located in the hierarchy
+`Activity.WorkFlowExecutionActivity`, but now it and 11 similar classes are located in
```

Comment:
I'm not familiar with this notation. Is `WorkflowExecutionActivity` an attribute of `Activity`? It reminds me of `instance.attribute` notation.

Reply:
I like this but want to discuss with others. I'm adding a "Conventions" line explaining this notation for now.